Clinical-Genomics/scout
# Fix #2245 - managed variants defaults to all categories

This PR adds a functionality or fixes a bug.

**How to test**:
1. try adding a couple of managed variants from different categories and make sure all are shown by default
2. also check that the multi select actually works buy checking a few different groups, single and no category and confirming result
3. consider if we should mark the checkboxes filled in on first load. May be more intuitive, while I think users will forgive us as no filtering is done when none is checked. And this is not sensitive. It may at one point become a long list, although thanks to ClinVar and the like - maybe not, but we can deal with that then.

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.


Managed variant:
<img width="1429" alt="Screen Shot 2021-06-11 at 11 05 18" src="https://user-images.githubusercontent.com/41540288/121661832-ea57c500-caa4-11eb-8fd6-56604e4c91df.png">


Category and Kind drop-down:
<img width="503" alt="Screen Shot 2021-06-11 at 10 17 21" src="https://user-images.githubusercontent.com/41540288/121662001-1ecb8100-caa5-11eb-810e-7944b6ec8640.png">
<img width="496" alt="Screen Shot 2021-06-11 at 10 17 31" src="https://user-images.githubusercontent.com/41540288/121662004-1ffcae00-caa5-11eb-9678-5d47db5a9409.png">
<img width="490" alt="Screen Shot 2021-06-11 at 10 17 40" src="https://user-images.githubusercontent.com/41540288/121662006-20954480-caa5-11eb-8dd6-26801f80480d.png">
<img width="499" alt="Screen Shot 2021-06-11 at 10 17 54" src="https://user-images.githubusercontent.com/41540288/121662008-20954480-caa5-11eb-936c-6bd14f1302bd.png">



**Review:**
- [x] tests executed by MOD
